aggregator_api/src/tests.rs

```diff
@@ -86,7 +86,8 @@ async fn get_config() {
         concat!(
             r#"{"protocol":"DAP-07","dap_url":"https://dap.url/","role":"Either","vdafs":"#,
             r#"["Prio3Count","Prio3Sum","Prio3Histogram","Prio3CountVec","Prio3SumVec"],"#,
-            r#""query_types":["TimeInterval","FixedSize"]}"#
+            r#""query_types":["TimeInterval","FixedSize"],"#,
+            r#""features":["token-hash"]}"#,
```

Comment:
nit: the rest of our features are camel case. No opinion on whether that's right or not, but I think it stands to be consistent.

Reply:
Sure, I changed it to `TokenHash`. @jbr do you have a strong feeling about the string here?